Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityConfigurationProfile.asset

```diff
@@ -25,3 +25,5 @@ MonoBehaviour:
   enableControllerProfiles: 1
   controllersProfile: {fileID: 11400000, guid: 4319257468937c74ebb69e41792d60c5, type: 2}
   enableBoundarySystem: 1
+  enableGraphAccessProfile: 1
```

Comment:
Do we want to enable this by default?

Reply:
Yeah, this file should be reverted, my mistake.

Reply:
I'm not sure I want to add this to the confirmation profile. In my pr I just assigned a profile from the project window onto my script that needed to know about graph.

I suppose this is something we can discuss later on.

Reply:
Sorry, I meant, Yes - you are right :) I should have reverted this file before pushing last changes, but I will do that next.